src/Components/Blazor/WebAssembly.Authentication/src/Services/RemoteAuthenticationService.cs

```diff
@@ -20,8 +21,13 @@ public class RemoteAuthenticationService<TRemoteAuthenticationState, TProviderOp
          where TRemoteAuthenticationState : RemoteAuthenticationState
          where TProviderOptions : new()
     {
+        private const int _userCacheRefreshInterval = 60;
         private bool _initialized = false;
 
+        // This defaults to 1/1/1970
+        private DateTimeOffset _userLastCheck = DateTimeOffset.FromUnixTimeSeconds(0);
```

Comment:
Would the following work? Avoids having to explain about 1/1/1970:

```suggestion
        private DateTimeOffset _userLastCheck = DateTimeOffset.MinValue;
```

Reply:
Don't care eitherway, FromUnixTimeSeconds is better IMO as that what the OIDC spec uses, but that is good to.